lib/python/treadmill_aws/ec2client.py

```diff
@@ -129,6 +129,10 @@ def get_image(ec2_conn, ids=None, tags=None, owners=None, name=None):
     images = list_images(
         ec2_conn, ids=ids, tags=tags, owners=owners, name=name
     )
+
+    if not images:
```

Comment:
if you look at latest code, we are using treadmill.exc.NotFoundError, treadmill.exc.FoundError everywhere. i am trying to streamline exception classes.